### PR TITLE
WIP -Playbook merger

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Comment by geoand** [comment]
This will have series of commits. 
I will remove the `WIP` designation once I'm done in order to indicate that it's ready for review/merge

File: ansible/playbook/roles/generate_inventory/templates/cloud.inventory.j2 [hunk 1]
```diff
@@ -1,56 +1,58 @@
...
 # For Atomic VM
-containerized=false
+containerized = false
 
 openshift_release=v3.7
 openshift_image_tag=v3.7.0
 openshift_pkg_version="-3.7.0"
+
 openshift_deployment_type=origin
-openshift_enable_service_catalog=true
```
Comment (cmoulliard) [inline ansible/playbook/roles/generate_inventory/templates/cloud.inventory.j2]: Is there a reason why you removed it as it is required for the lab ? should we enable/disable base on a property passed when we generate the inventory ? @geoand

Comment (geoand) [reply ansible/playbook/roles/generate_inventory/templates/cloud.inventory.j2]: Actually I haven't removed it, I just changed it's position in the file to match where it is in the playbook that exists under `virtualbox`. See this: https://github.com/snowdrop/cloud-native-infra/blob/365b049f557ee32f6350aac53ed8e7eb6009dbdc/ansible/playbook/roles/generate_inventory/templates/cloud.inventory.j2#L36

**Comment by cmoulliard** [comment]
Do you want that I merge it ?

**Comment by geoand** [comment]
Not yet, I want to add some more commit as well, so we can review them as a whole :)

**Comment by geoand** [comment]
@cmoulliard This should be ready for review.
I removed all Ansible related functionality from the `virtualbox` directory and updated the playbooks in the `ansible` to reflect your latest changes.

**Comment by geoand** [comment]
I have updated the PR to consolidate the latest changes from master

**Pull request merged**